dotNETStandard/HealthVaultProviderManagementPortal/HealthVaultProviderManagementPortal/Controllers/PatientController.cs

```diff
@@ -63,6 +69,28 @@ public class PatientController : Controller
             return View(model);
         }
 
+        [HttpGet]
+        public async Task<ActionResult> TaskOccurrence(Guid personId, Guid recordId, Guid id)
+        {
+            var response = await ExecuteMicrosoftHealthVaultRestApiAsync(api => api.ActionPlanTasks.GetByIdAsync(id.ToString()), personId, recordId);
+            return View(response);
+        }
+
+        [HttpPost]
+        public async Task<ActionResult> TaskOccurrence(Guid personId, Guid recordId, Guid id, DateTimeOffset trackingDateTime, DateTime? startDate, DateTime? endDate)
```

Comment:
It's there so that I can redirect back to the timeline view that you came from. No other reason.